ci/installdeps.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -xeuo pipefail
+
+yum -y install skopeo
+yum -y --enablerepo=updates-testing update ostree-devel
```

Comment:
Hmm true but OTOH I think let's keep it because I've actually had a new desire to add new API to ostree again, and while this still requires an ostree release, we don't need to block on bodhi at least.